lib/src/control_panel/boolean_control.dart

```diff
@@ -25,7 +27,10 @@ class BooleanControl extends StatelessWidget {
   Widget build(BuildContext context) {
     return Row(
       children: <Widget>[
-        title,
+        SizedBox(
+          width: EnvironmentControlPanel.labelWidth,
```

Comment:
Why do these labels need to individually have their width limited? Would it be possible to just put the entire EnvironmentControlPanel into a SizedBox with a fixed width, and then have the layout system deal with it?

Reply:
This is a way to align text between child widgets of the control panel. I'll add before/after screenshots to the PR description to illustrate.